eslint.config.mjs

```diff
@@ -0,0 +1,10 @@
+import neolutionEslintConfig from "@neolution-ch/eslint-config-neolution";
+import storybook from "eslint-plugin-storybook";
+
+export default [
+  {
+    ignores: ["**/cypress/"],
+  },
+  ...neolutionEslintConfig.configs.flat.nextjs,
```

Comment:
@neotob 

this should be `.flat["react-library"]` right?

Reply:
100% right, many thanks :)